src/configure.py

```diff
@@ -0,0 +1,1000 @@
+            indent     = "Usage: %s " % os.path.basename(sys.argv[0])
+            output = indent + " " + str.join(" ", usage)
+            print( self.string_align(output, indent, usage_width, "]") )
+        print("")
+
+    def print_help(self, *args, **kwargs):
+        # Print usage
+        self.print_usage()
+
+        # Print description
+        if "description" in self.program:
+            print(self.program["description"])
+            print("")
+
+        # Print epilog
+        if "epilog" in self.program:
+            print(self.program["epilog"])
+            print("")
+
+    def print_help_detail(self):
+        # Print usage
+        self.print_usage()
+
+        # Print description
+        if "description" in self.program:
+            print(self.program["description"])
+            print("")
+
+        # Print options
+        print("Options:")
+        option_width = 100
+        option_indent = 0
+        for option in self.options:
+            option["flags2"] = str.join(", ", [ "%s %s" % (item, option["metavar"]) if "metavar" in option else "%s %s" % (item, option["dest"].upper()) if "dest" in option else item for item in option["flags"] ])
+            if len(option["flags2"]) > option_indent:
+                option_indent = len(option["flags2"])
+
+        for option in self.options:
+            template = "  %-" + str(option_indent) + "s  "
+            indent = template %(option["flags2"])
+            output = indent
+
+            if "help" in option: output += option["help"]
+
+            if "action" in option:
+                if option["action"] == "help":
+                    print( self.string_align(output, indent, option_width, " ") )
+                    continue
+
+            if "choices" in option:
+                temp = [ str(opt) for opt in option["choices"] ]
+                output += "Choice: [%s] => "%(", ".join(temp))
+
+            if "default" in option:
+                output += "Default: %s" % option["default"] if isinstance(option["default"], str) else "Default: %s" % str(option["default"])
+
+            if "action" in option:
+                output += "Default: False" if option["action"] == "store_true" else "Default: False"
+
+            print( self.string_align(output, indent, option_width, " ") )
+
+        # Print epilog
+        if "epilog" in self.program:
+            print(self.program["epilog"])
+            print("")
+
+
+
+####################################################################################################
+# Functions
+####################################################################################################
+def color_print( string, color ):
+    print( color + string + BCOLOR.ENDC )
+    return
+
+def add_option( opt_str, name, val ):
+    # NOTE: every -Doption must have trailing space
+    if type(val) == type(True):
+        if val:
+            opt_str += "-D%s "%(name)
+            print("%-20s : %r"%(name, val))
```

Comment:
- [ ] How about also printing options that are disabled? For example,
```
        if val: opt_str += "-D%s "%(name)
        print("%-20s : %r"%(name, val))
```
- [ ] Increase `20` to `25` (modify the lines below too).